ironfish-rust-nodejs/tests/demo.test.slow.ts

```diff
@@ -29,10 +29,16 @@ describe('Demonstrate the Sapling API', () => {
 
   it('Should return phrase words from key input', () => {
     const key = generateKey()
```

Comment:
Same thing with this test, you could make it deterministic if you wanted with `generateKeyFromPrivateKey` which takes a hex string. It might be worth writing tests here and in the rust to do the "full flow" of starting at 1 point, say, a key, going to words, then back to the key and asserting its the same as well. I guess if you have both tests using deterministic inputs its essentially the same thing.